src/pages/landing/ebooks/forex.js

```diff
@@ -30,16 +31,27 @@ const topicsCovered = [
 
 const query = graphql`
     query {
-        forex_ebook_img: file(relativePath: { eq: "ebooks/forex-ebook.png" }) {
+        forex_ebook_img_en: file(relativePath: { eq: "ebooks/forex-ebook-en.png" }) {
             ...bannerImage
         }
-        forex_ebook_inside: file(relativePath: { eq: "ebooks/forex-ebook-inside.png" }) {
+        forex_ebook_img_es: file(relativePath: { eq: "ebooks/forex-ebook-es.png" }) {
+            ...bannerImage
+        }
+        forex_ebook_inside_en: file(relativePath: { eq: "ebooks/forex-ebook-inside-en.png" }) {
+            ...bannerImage
+        }
+        forex_ebook_inside_es: file(relativePath: { eq: "ebooks/forex-ebook-inside-es.png" }) {
             ...bannerImage
         }
     }
 `
 
 const ForexEbook = () => {
+    const { i18n } = useTranslation('home')
```

Comment:
Can we get the language from the props available in ForexEbook component instead to reduce imports?...should be available in `props.language`

Reply:
You can also get it from `localStorage.getItem('i18n')`.

Reply:
done thanks.